### ISSUE TITLE
QUnit / OPA tests are generating console errors because of invalid use of `QUnit.config.autostart = false;`

### ISSUE THREAD

**Issue by ArnaudBuchholz** [ui5-community/generator-ui5-app#10]
## Easy-Ui5 version:

Latest version

## OS/version:

Not relevant

## Browser/version (+device/version):

Not relevant

## Steps to reproduce the problem:

1. Generate an static server app with `easy-ui5`
2. Serve the newly created app
3. Open the qunit or OPA tests

## What is the expected result?

Tests are executed (and they are)

## What happens instead?

Depending on the server used, the The console log shows an error about QUnit being called on start while the tests are already started.
![image](https://github.com/ui5-community/generator-ui5-project/assets/4194952/9e3bbd91-8958-4525-82eb-1dd49ac7b295)

## Any other information? (attach screenshot if possible)

The problem comes from the asynchronous loading of the `unitTests.qunit` module that happens asynchronously *after* qunit is loaded. Hence it does not prevent the startup soon enough.

https://github.com/ui5-community/generator-ui5-app/blob/95ee595678d06576ccd6a3732349e158bd987796/generators/app/templates/webapp/test/unit/unitTests.qunit.html#L17

I would recommend inserting a small script file that *synchronously* prevents the tests to start like in :
https://github.com/ArnaudBuchholz/training-ui5con18-opa/blob/main/webapp/test/qunit.autostart.js

If you agree with it, I can submit a PR.

**Comment by ArnaudBuchholz** [comment]
Indeed, test starter is a nice replacement which doesn't cause this issue. You may close the issue.

**Comment by akudev** [comment]
Closing, as the issue does no longer occur for newer versions and is not a real problem for older ones.
